capstone/src/main/webapp/js/autocomplete.js

```diff
@@ -88,6 +88,32 @@ function closeAllElements() {
   }
 }
 
+/**
+ * Keeps track of chosen colleges
+ * @param {Array<string, collegeId>} validSuggestion
+ */
+function keepTrackOfChosenColleges(validSuggestion) {
+  const parent = document.getElementById('suggestions');
+  const listElement = document.createElement('li');
+  listElement.setAttribute('id', validSuggestion.collegeId.id);
+  listElement.innerHTML = validSuggestion.collegeName;
+  listElement.innerHTML += ' ';
+  const removeButton =
+    '<button onclick=\'removeCollege('+
+    validSuggestion.collegeId.id + ')\'>X</button';
```

Comment:
The X may not render as nicely as we want, since it won't show up like ❌ or ✖️ or anything like that. It will just show up as text. I would suggest either using a special Unicode emolji/character, or just use the string "Remove" instead of "X".

Reply:
![image](https://user-images.githubusercontent.com/43079005/91897729-95526d00-ec4f-11ea-912a-75f8ee1fa413.png)
This is how it renders right now. I think in addition to replacing the "X", the button should appear below the university name.

Reply:
done :)